rpm/droidmedia.spec

```diff
@@ -46,7 +46,12 @@ mv droidmedia* droidmedia
 popd
 
 %build
+
+%if "%{?BOARD_QTI_CAMERA_32BIT_ONLY:%{BOARD_QTI_CAMERA_32BIT_ONLY}}%{!?BOARD_QTI_CAMERA_32BIT_ONLY:0}" == "true" 
```

Comment:
I may have forgotten how to read this line, but all in all, I don't think AOSP defined env macros are available in the Platform SDK (mb2) realm, so .spec wouldn't see them. Please correct if I'm wrong :)